web/src/jobs/process_article.rs

```diff
@@ -25,7 +26,30 @@ impl Job<AppState> for ProcessArticle {
             .await
             .into_diagnostic()?;
 
-        persist_article(db, page).await.unwrap();
+        let raw_html = download_raw_html(&page.url).await.unwrap();
+        let url_parsed = Url::parse(&page.url).unwrap();
+        let current_time = chrono::Utc::now();
+        let cleaned_html = clean_raw_html(&raw_html, &url_parsed).unwrap();
+
+        let page_snapshot = sqlx::query_as!(
```

Comment:
Attempt to insert a new page snapshot into the database and return the inserted record.